db/influx.go

```diff
@@ -236,7 +230,8 @@ func (i *influxDB) TodaysLatency() (map[string][]api.Latency, error) {
 				}
 
 			}
-			latencyByHour := api.Latency{Time: hourlyTime, Latency: roundFloat(hourlyAverageLatency, 5)}
+
+			latencyByHour := api.Latency{Time: hourlyTime, Latency: numbers.RoundFloat(hourlyAverageLatency, 5)}
```

Comment:
Please use GO's math library for rounding: https://pkg.go.dev/math

Reply:
resolving per our convo on discord